### PR TITLE
Migrate More Admin Reports to B3 (fix indicator reports)

### PR DESCRIPTION
**Pull request by biyeun**
fixes indicator admin reports (currently look wonky...sorry). migrates all admin faceted reports to B3

@sravfeyn 
cc interrupt @calellowitz @benrudolph @kaapstorm 

screen shot 2016-01-04 at 4 11 35 pm
screen shot 2016-01-04 at 4 12 13 pm
screen shot 2016-01-04 at 4 24 03 pm
screen shot 2016-01-04 at 4 24 17 pm
screen shot 2016-01-04 at 4 24 33 pm

### CODE REVIEW

File: corehq/apps/hqadmin/reports.py [hunk 1]
```diff
@@ -550,7 +557,7 @@ class AdminFacetedReport(AdminReport, ElasticTabularReport):
     es_facet_mapping = []
     section_name = ugettext_noop("ADMINREPORT")
     es_url = ''
-
+
```
Comment (dimagimon) [inline corehq/apps/hqadmin/reports.py]: W293 blank line contains whitespace

File: corehq/apps/hqadmin/reports.py [hunk 2]
```diff
@@ -640,14 +647,9 @@ class AdminDomainStatsReport(AdminFacetedReport, DomainStatsReport):
...
 
-    report_template_path = "reports/async/bootstrap3/tabular.html"
-
-    @use_bootstrap3
-    @use_datatables
-    @use_jquery_ui
     @use_nvd3
     def set_bootstrap3_status(self, request, *args, **kwargs):
-        self.is_bootstrap3 = True
+        super(AdminDomainStatsReport, self).set_bootstrap3_status(request, *args,**kwargs)
```
Comment (dimagimon) [inline corehq/apps/hqadmin/reports.py]: E231 missing whitespace after ','

**Comment by benrudolph** [comment]
:+1: after lint and tests

**Comment by sravfeyn** [comment]
g2m by me. Build fail seems unrelated. Kicking again.

**Comment by kaapstorm** [comment]
:+1:

**Comment by kaapstorm** [comment]
:+1:

**Pull request merged**